src/Context/ApiContext.tsx

```diff
@@ -0,0 +1,17 @@
+
+     import { createContext } from 'react'
+
+     export const ApiContext = createContext(null);
+
+     export default function ApiContextProvider (props:any) {
+
+     let baseUrl = 'https://upskilling-egypt.com:3000/api/v0'
```

Comment:
I suggest creating .env file to store the base URL. This will be better if one of our endpoints depends upon a paid API. This means that we are making it available for everyone for free and we pay the bill.